src/pages/reports/index.jsx

```diff
@@ -28,18 +28,31 @@ const ReportsBreadcrumb = ({ category }) => {
   );
 };
 
+const SectionBanner = ({ category }) => {
+  let title = <h2>Informes</h2>;
+  if (category) {
+    title = <h2>{category.title}</h2>;
+  }
```

Comment:
This could be done via this instead, which I prefer to the stateful `let` declaration:

```js
const title = (<h2>{category ? category.title : "Informes"}</h2>
```

This could also be inlined below, such as this:

```js
return (
    <div className="section-header">
      <h2>{category ? category.title : "Informes"}</h2>
    </div>
  );
```

And then you can make the function return implicitly.